test/interceptors/cache.js

```diff
@@ -1489,47 +1492,55 @@ describe('Cache Interceptor', () => {
     })
   }
 
-  test('does not cache non-heuristically cacheable error status codes', async () => {
```

Comment:
The RFC allows to cache non-heuristically cacheable error responses if they have an explicit caching directive.

So, I change this test to check we do not cache non-heuristically cacheable response not having a caching directive.